README.md

```diff
@@ -53,11 +53,12 @@ This HoC will provide `Component` with 3 props:
 `value`: The current value for the input  
 `setValue`: A callback function which replace the existing form value 
 `error`: The current error message associated with the input, or null
+`inputProps`: An object of meta values which are passed to all inputs within the form
```

Comment:
@shvagen Can you update the `Form` section as well to indicate that users define `inputProps` by passing it in as a prop into `Form`?